### PR TITLE
[#70] Review and fix serialized Did Document representation

### PR DESCRIPTION
**Pull request by tdiesler**

### CODE REVIEW

**Comment by victormartinez-work** [comment]
Hi, @tdiesler thanks for your contribution and your fantastic work adding SSI capabilities to apache camel!

File: lib/src/test/kotlin/org/didcommx/didcomm/diddoc/DIDDocEncodingTest.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*-
+ * #%L
+ * Nessus DIDComm :: Core
```
Comment (victormartinez-work) [inline lib/src/test/kotlin/org/didcommx/didcomm/diddoc/DIDDocEncodingTest.kt:3]: Let's remove references to [https://github.com/tdiesler/nessus-didcomm](nessus-didcomm)

**Review by victormartinez-work — CHANGES_REQUESTED** [review]

File: lib/src/test/kotlin/org/didcommx/didcomm/diddoc/DIDDocEncodingTest.kt (hunk 1 above)
Comment (tdiesler) [reply lib/src/test/kotlin/org/didcommx/didcomm/diddoc/DIDDocEncodingTest.kt]: Thanks ... header removed

**Review by victormartinez-work — APPROVED** [review]

**Comment by tdiesler** [comment]
Now contained in https://github.com/sicpa-dlab/didcomm-jvm/pull/77

**Pull request closed without merging**